src/game-engine/world/map/chunk-manager.ts

```diff
@@ -7,27 +7,27 @@ import { LandscapeFile, LandscapeObject, MapFile } from '@runejs/filestore';
 
 export class Tile {
 
-    public settings: number;
-    public nonWalkable: boolean;
-    public bridge: boolean;
+    public settings: number = 0;
+    public blocked: boolean = false;
+    public bridge: boolean = false;
 
     public constructor(public x: number, public y: number, public level: number, settings?: number) {
-        if(settings !== undefined) {
+        if(settings) {
```

Comment:
should this if statement run if settings == 0? because doing `if (settings)` will return false if settings == 0.

Reply:
A settings value of zero is a standard walkable tile, so no need to check it :)